### PR TITLE
detox package addition

### PR DESCRIPTION
**Pull request by dmzoneill**

### CODE REVIEW

File: cross/detox/Makefile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+PKG_NAME = detox
+PKG_VERS = 1.2.0
+PKG_VERS_CUT = -1.2.0
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = ftp://ftp.heanet.ie/mirrors/download.sourceforge.net/pub/sourceforge/d/de/detox/detox/1.2.0
+PKG_DIR = $(PKG_NAME)$(PKG_VERS_CUT)
+
+#DEPENDS = cross/ncurses
```
Comment (cytec) [inline cross/detox/Makefile]: if this isn't needed just change it to `DEPENDS = ` instead of commenting it out

File: spk/detox/src/installer.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put mc in the PATH
```
Comment (cytec) [inline spk/detox/src/installer.sh]: copy paste leftover?

File: spk/detox/src/installer.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    mkdir -p /usr/local/bin
+    mkdir -p /usr/local/usr/share/detox
```
Comment (cytec) [inline spk/detox/src/installer.sh]: not sure if all this is needed... have you tried just putting detox in path? like mc does (https://github.com/SynoCommunity/spksrc/blob/master/spk/mc/src/installer.sh#L24)

**Comment by hgy59** [comment]
detox is merged to master with #3776

**Pull request closed without merging**